pkgs/development/libraries/libgpiod/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit, autoreconfHook, autoconf-archive, pkgconfig, kmod, enable-tools ? true }:
+
+stdenv.mkDerivation rec {
+  name = "libgpiod-${version}";
+  version = "2018-10-07";
```

Version should be:
```suggestion
  version = "unstable-2018-10-07";
```
Per the package naming guidelines:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".